temporian/implementation/numpy_cc/operators/calendar/day_of_year.cc

```diff
@@ -0,0 +1,49 @@
+#include <pybind11/numpy.h>
+#include <pybind11/pybind11.h>
+
+#include <cstdint>
+#include <ctime>
+#include <iostream>
+#include <map>
+#include <string>
+#include <vector>
+
+#include "absl/time/time.h"
+#include "temporian/implementation/numpy_cc/operators/calendar/common.h"
+#include "temporian/implementation/numpy_cc/operators/common.h"
+
+namespace {
+namespace py = pybind11;
+
+py::array_t<int> calendar_day_of_year(const py::array_t<double> &timestamps,
```

Comment:
A large part of this code is similar in the different operators. Would it be possible to factorize it?

Reply:
Please comment on action.